### ISSUE TITLE
Installation under different username --> permissions problem?

### ISSUE THREAD

**Issue by bertmelis** [billw2/pikrellcam#36]
After a successful installation on a fresh Raspbian Stretch (Lite) on a RPI1B, Pikrellcam starts nicely. However, I don't get any image. I've found the following in the logs:
```
make_dir(/run/pikrellcam) failed. No such file or directory
Failed to create media directories or FIFO, exiting!
```
The camera module works as `raspistill` generates an image.

**Comment by bertmelis** [comment]
ah wait, I installed it on my user account, and not `pi`. Is that supported?

**Comment by billw2** [comment]
The user installing/running pikrellcam needs to be able to change owner/modes of directories so
that the web server www-data has permission to read/write files.  At least the user needs sudo
privilege and permission to access the /run directory.  Maybe you can set up another user
to work, but I haven't done it so can't give you the set of steps you would need.

**Comment by bertmelis** [comment]
I always remove the `pi` user from the system and replaces it with my own with sudo and video privileges. But Pikrellcm won't display any image.

To troubleshoot I created the `pi` user again, reran the installation script from this user and now everything worked!

The thing I did different was that I had autostart at boot enabled the first time and I disabled it when I installed again with the `pi`-user.
